libs/blocks/caas-config/caas-config.js

```diff
@@ -377,6 +377,10 @@ const TagsPanel = ({ tagsData }) => {
     });
   };
 
+  const secondarySourcePanel = html`
+    <${DropdownSelect} options=${defaultOptions.source} prop="secondSource" label="Secondary Source" />
```

Comment:
Minor (subjective) edit: 
For consistency, should this be prop be named `secondarySource`?

Reply:
since the chimera-io changes accepting "secondSource" have already merged I am going to push back on changing this to be "secondarySource" to avoid having to change the property value downstream.  But this is a good point and I probably should've named it that all along